.github/workflows/e2e-test.yml

```diff
@@ -0,0 +1,35 @@
+# Run nri resource policy plugin e2e tests
+
+name: NRI Resource Policy CI
+
+# Controls when the workflow will run
+on:
+  pull_request:
+    branches: [ "main" ]
+#  schedule:
+#    - cron: '0 12 * * *'
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+jobs:
+  e2e-tests:
+    runs-on: self-hosted
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v3
+
+      - name: Update vendoring data
+        run: go mod vendor
+
+      - name: Build binaries
+        run: make
+
+      - name: Build images
+        run: make images
+
+      - name: Run topology-aware e2e tests
+        run: test/e2e/run_ci.sh policies.test-suite/topology-aware/n4c16/
```

Comment:
```suggestion
      - name: Run topology-aware e2e tests
        run: ./test/e2e/run_ci.sh test/e2e/policies.test-suite/topology-aware/n4c16/
```

Reply:
This is actually not needed because the run_ci.sh script will change its directory to test/e2e and run the run_tests.sh from there.